blockonomics-woocommerce.php

```diff
@@ -223,122 +218,232 @@ function show_basic() {
+                                <th scope="row"><?php echo __('Display Payment Page in Lite Mode (Enable this if you are having problems in rendering checkout page)', 'blockonomics-bitcoin-payments')?></th>
+                                <td><input type="checkbox" name="blockonomics_lite" value="1" <?php checked("1", get_option('blockonomics_lite')); ?> /></td>
+                            </tr>
+                            <tr valign="top">
+                                <th scope="row"><?php echo __('No Javascript checkout page (Enable this if you have majority customer that use tor like browser that block Javascript)', 'blockonomics-bitcoin-payments')?></th>
+                                <td><input type="checkbox" name="blockonomics_nojs" value="1" <?php checked("1", get_option('blockonomics_nojs')); ?> /></td>
+                            </tr>
+                            <tr valign="top">
+                                <th scope="row"><?php echo __('Network Confirmations required for payment to complete)', 'blockonomics-bitcoin-payments')?></th>
+                                <td><select name="blockonomics_network_confirmation" />
+                                        <option value="2" <?php selected(get_option('blockonomics_network_confirmation'), 2); ?>>2 (Recommended)</option>
+                                        <option value="1" <?php selected(get_option('blockonomics_network_confirmation'), 1); ?>>1</option>
+                                        <option value="0" <?php selected(get_option('blockonomics_network_confirmation'), 0); ?>>0</option>
+                                    </select></td>
+                            </tr>
+                        </table>
+                    </div>
+                    <p class="submit">
+                        <input type="submit" class="button-primary" value="Save"/>
+                        <input type="hidden" name="action" value="update" />
+                        <input type="hidden" name="page_options" value="blockonomics_api_key,blockonomics_timeperiod,blockonomics_margin,blockonomics_gen_callback,blockonomics_api_updated,blockonomics_underpayment_slack,blockonomics_lite,blockonomics_nojs,blockonomics_network_confirmation" />
+                    </p>
+                </form>
+                <form method="POST" name="generateSecretForm">
+                        <p class="submit">
+                            <input type="hidden" name="generateSecret" value="true">
+                        </p>
+                </form>
+                    <?php
+                    break;
+
+
+
+
+                case 'currencies' :
+                    ?>
+                    <table style="margin-left:40px; padding: 0px 0px !important; " class="form-table">
+                    <h1>
+                        <input style="margin-left:12px;" type="checkbox" name="blockonomics_btc" value="1"<?php checked("1", get_option('blockonomics_btc')); ?>" />
```

Comment:
Bitcoin should be enabled by default if not set yet- `get_option("blockonomics_btc", true);`

Reply:
Made and committed the change.